assets/test/automation/tests/tqa-6.spec.ts

```diff
@@ -0,0 +1,57 @@
+import { test, expect } from "@playwright/test";
+
+test("test", async ({ page }) => {
+  await page.goto("https://stellarator.oli.cmu.edu/");
```

Comment:
This URL should be externalized into an environment variable - as we need to be able to run these automated tests against different servers.